src/main/scala/mesosphere/marathon/api/v2/EventSubscriptionsResource.scala

```diff
@@ -23,18 +23,16 @@ import scala.concurrent.Future
 class EventSubscriptionsResource @Inject() (
     val config: MarathonConf,
     val authenticator: Authenticator,
-    val authorizer: Authorizer) extends AuthResource {
-
-  //scalastyle:off null
-
-  @Inject(optional = true) val service: HttpCallbackSubscriptionService = null
+    val authorizer: Authorizer,
+    val maybeService: Option[HttpCallbackSubscriptionService]) extends AuthResource {
```

Comment:
Instead of having an `Option[HttpCallbackSubscriptionService]` can we use `HttpCallbackSubscriptionService` with a different implementation (e.g. noop)? So client code does not have to deal with the existence of the service?